deepfence_utils/directory/minio.go

```diff
@@ -141,16 +141,26 @@ func (mfm *MinioFileManager) ListFiles(ctx context.Context, pathPrefix string, r
 	return objectsInfo
 }
 
-func (mfm *MinioFileManager) UploadLocalFile(ctx context.Context, filename string, localFilename string, extra interface{}) (UploadResult, error) {
+func (mfm *MinioFileManager) UploadLocalFile(ctx context.Context,
+	filename string, localFilename string, overwrite bool, extra interface{}) (UploadResult, error) {
+
 	err := mfm.createBucketIfNeeded(ctx)
 	if err != nil {
 		return UploadResult{}, err
 	}
 
 	objectName := mfm.addNamespacePrefix(filename)
 
-	if key, has := checkIfFileExists(ctx, mfm.client, mfm.bucket, objectName); has {
-		return UploadResult{}, AlreadyPresentError{Path: key}
+	key, has := checkIfFileExists(ctx, mfm.client, mfm.bucket, objectName)
+	if has {
+		if !overwrite {
+			return UploadResult{}, AlreadyPresentError{Path: key}
+		} else {
+			err := mfm.DeleteFile(ctx, objectName, false, minio.RemoveObjectOptions{ForceDelete: true})
```

Comment:
I think we should add a log here that we have found an existing file and will be attempting to overwrite this. 
This would come handy during debugging.